### PR TITLE
Fix ru_Ru.lang

### PR DESCRIPTION
**Pull request by kamenskyi**
fix ru_Ru.lang с файла перевода офицальный на версию gtnh-7.6

### CODE REVIEW

**Comment by kamenskyi** [comment]
Баг не подхватывается перевод в определенных местах при игре. Wand, cloth, wand focus

File: src/main/resources/assets/taintedmagic/lang/ru_Ru.lang [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+#=========ITEM NAMES===================
+
+item.ItemMaterial.0.name=Слиток Мрачного Металла
+item.ItemMaterial.1.name=Мрачная ткань
+item.ItemMaterial.2.name=Багровая ткань
+item.ItemMaterial.3.name=Искаженный кристалл
+item.ItemMaterial.4.name=Заражённый кристалл
+
+item.ItemMaterial.5.name=Осколок Мироздания
+text.shard.=Ключ к тому, как все это произошло...
```
Comment (repo-alt) [inline src/main/resources/assets/taintedmagic/lang/ru_Ru.lang:54]: точка на конце text.shard, может не найтись строка

File: src/main/resources/assets/taintedmagic/lang/ru_Ru.lang [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+
+#===========RESEARCH CATEGORY============
+
+tc.research_category.TM=Магия Порчи
+
+#===========THAUMONOMOCON TEXT===========
+
+#TAINTEDMAGIC
+
+tm.name.TAINTEDMAGIC= Тревожная Находка
```
Comment (repo-alt) [inline src/main/resources/assets/taintedmagic/lang/ru_Ru.lang:143]: насколько я помню вокруг = не должно быть пробелов

File: src/main/resources/assets/taintedmagic/lang/ru_Ru.lang [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+
+#ELDRITCHFOCUS
+
+tm.name.ELDRITCHFOCUS=Набалдашник: Тёмная материя
+tm.tag.ELDRITCHFOCUS="Чипух" (c) Quineros 
+tm.text.ELDRITCHFOCUS.1=Используя свои знания о набалдашниках и первородных энергиях, что положили начало всему...<BR>Использовав жемчужину, для создания тех кристаллов... Потянув за ниточки мироздания, вы создаете мощное оружие, чтобы управлять тёмной материей, как некогда Древние.<BR>§oЯ вглядываюсь в полированную поверхность набалдашника и думаю - что же со мной стало? Меня ничто не остановит... §r§lНИЧТО!<BR>(Continued)
+tm.text.ELDRITCHFOCUS.2=Поскольку этот набалдашник наполнен пробудившейся силой Осколка Творения, шары оказались даже силнее тех, которыми в вас стреляют стражи. Они наносят 5 сердец урона и при контакте накладывают зелье слабости.<BR>Однако при использовании этого набалдашника следует соблюдать осторожность. Каждый раз когда вы фокусируетесь и вызываете сферу есть шанс, что сферазаставит вас получить небольшое количество искажения... Но небольшое искажение не кому не повредит, верно?
```
Comment (repo-alt) [inline src/main/resources/assets/taintedmagic/lang/ru_Ru.lang:209]: сферазаставит

**Comment by repo-alt** [comment]
(as for the incomplete translation support - this fork was made from a very old version of TM, it had tons of updates since. Raven tried to rebase our branch on the latest upstream but looks like didn't finish yet)

**Comment by Alastors** [comment]
No we already had

**Comment by Alastors** [comment]
Chocho did it awhile back with latest history, we're effectively making a separate mod out of a fork of 7.6

**Comment by repo-alt** [comment]
Ah, ok then.

**Review by repo-alt — APPROVED** [review]

**Pull request merged**